WiserTaskScheduler/WiserTaskScheduler/Core/Models/WtsSettings.cs

```diff
@@ -4,6 +4,8 @@
 using WiserTaskScheduler.Core.Models.Cleanup;
 using WiserTaskScheduler.Core.Models.ParentsUpdate;
 using WiserTaskScheduler.Core.Services;
+using WiserTaskScheduler.Modules.Branches.Models;
+using WiserTaskScheduler.Modules.Branches.Services;
```

Comment:
Did the using for the services get added accidentally?

Reply:
No, they are needed for things like `<see cref="AutoProjectDeployService"/>` in the documentation of the properties in this class.